static/app/data/forms/projectIssueGrouping.tsx

```diff
@@ -97,6 +97,34 @@ stack.function:mylibrary_* +app`}
     validate: () => [],
     visible: true,
   },
+  derivedGroupingEnhancements: {
+    name: 'derivedGroupingEnhancements',
+    type: 'string',
+    label: 'Derived Grouping Enhancements (super user only)',
+    hideLabel: true,
+    placeholder: '',
+    multiline: true,
+    monospace: true,
+    autosize: true,
+    inline: false,
+    maxRows: 20,
+    saveOnBlur: false,
+    saveMessageAlertType: 'info',
+    saveMessage: '',
+    formatMessageValue: false,
+    help: () => (
+      <Fragment>
+        <RuleDescription>
+          These rules are automatically derived for some languages for customers that have
+          the GitHub integration and the language has been marked to derive in-app rules.
+          These rules are not editable but they can be negated by adding their own rules
+          in the Stack Trace Rules section.
+        </RuleDescription>
+      </Fragment>
```

Comment:
fragment not doing anything here
```suggestion
        <RuleDescription>
          These rules are automatically derived for some languages for customers that have
          the GitHub integration and the language has been marked to derive in-app rules.
          These rules are not editable but they can be negated by adding their own rules
          in the Stack Trace Rules section.
        </RuleDescription>
```